test/mock-rippled.ts

```diff
@@ -49,8 +48,17 @@ function createLedgerResponse(request, response) {
   return JSON.stringify(newResponse);
 }
 
-module.exports = function createMockRippled(port) {
-  const mock = new WebSocketServer({ port: port });
+interface CustomWebSocketServer extends WebSocketServer, EventEmitter2 {
+  expectedRequests: any
```

Comment:
Why create an interface for this? It's basically untyped anyway. 

I also don't know why it needs to extend anything.

Reply:
+1, cleaned this up and added a comment